webapp/pom.xml

```diff
@@ -434,27 +434,6 @@
                 </executions>
             </plugin>
 
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-surefire-plugin</artifactId>
-                <configuration>
-                    <!-- Sets the VM argument line used when unit tests are run. -->
-                    <argLine>-Dspring.config.additional-location=file://${user.home}/.l10n/config/webapp/
-                        -Dspring.profiles.active=${user.name},test -Xmx1024m -Duser.timezone=UTC
```

Comment:
- the memory and timezone should be at the very least kept
- what's the "new" way of doing what was done with `spring.config.additional-location`? ideally we should replace it, not remove it.

Reply:
The recommended way for files that might not exist is to use the "optional:" qualifier in the file path https://docs.spring.io/spring-boot/docs/current/reference/html/features.html

This configuration should be kept - and optional should be used on the path.

Reply:
agree let's try to use: `optional:-Dspring.config.additional-location=file://${user.home}/.l10n/config/webapp/...`

Reply:
*or -Dspring.config.additional-location=optional:file://${user.home}/.l10n/config/webapp/...
(whatever the syntax is)